singleton-service/src/configuration/service.rs

```diff
@@ -36,7 +33,6 @@ impl Default for ServiceConfig {
     fn default() -> Self {
         ServiceConfig {
             threescale_cluster: "threescale_SM_API".to_owned(),
-            threescale_auth_basepath: "authorize.xml".to_owned(),
```

Comment:
You'll still want a URL to fill in the Host header when Envoy calls this cluster - that URL will define as well any base path in which the API is mounted. You can find an example definition at https://github.com/3scale/threescale-wasm-auth/blob/88b81ba92291a68756393a3a8ce2b71e4acfc20a/src/upstream.rs#L11-L17 and usage within the same file with `call` and `call_url` methods.

Reply:
yeah, i didn't try to add the implementation here because this is just a skeleton for us to start working. I will add that http call logic in the next PR. So I won't have a  `threescale_auth_basepath` in the configuration but will have a URI like the auth filter and use that to perform the http call.